ShopAppTests/App/Account/Controllers/BaseAddressListViewControllerSpec.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  BaseAddressListViewControllerSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 3/2/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class BaseAddressListViewControllerSpec: QuickSpec {
+    override func spec() {
```

Comment:
Can we test `BaseAddressListControllerDelegate`?

Reply:
We test it on the bottom of spec